frontend/src/libs/components/button/button.tsx

```diff
@@ -6,21 +6,24 @@ import styles from './styles.module.scss';
 
 type Properties = {
   disabled?: boolean;
-  label: string;
+  label: string | React.ReactNode;
```

Comment:
ReactNode combines multiple times and string is one of them https://reactnative.dev/docs/react-node.
So you can simply do this:
```suggestion
  label: React.ReactNode;
```